.github/workflows/main.yaml

```diff
@@ -29,6 +29,7 @@ jobs:
         shell: bash -l {0}
         run: |
           pip install -e . --no-deps
+          pip install dask fsspec python-snappy
```

Comment:
Shouldn't these be in the environment-*.yml files?

Reply:
yeah that makes more sense